corehq/apps/reports/standard/cases/basic.py

```diff
@@ -107,7 +109,8 @@ def _build_query(self):
     @memoized
     def es_results(self):
         try:
-            return self._build_query().run().raw
+            with self.profiler.timing_context("ES query") if self.profiler_enabled else contextlib.nullcontext():
+                return self._build_query().run().raw
```

Comment:
nit: for sake of code clarity can this be extracted in a method and decorated with profile as well?

Reply:
https://github.com/dimagi/commcare-hq/pull/35830/commits/58576de8e14eb4297cba678c6e751357a2efc3a3